examples/ads.amp.html

```diff
@@ -1420,6 +1421,12 @@ <h2>RevJet Tag</h2>
       data-opts='{"delivery_method": "banner"}'>
   </amp-ad>
 
+  <h2>Red for Publishers</h2>
+  <amp-ad width="320" height="150"
```

Comment:
I found this ad slot always return me a no fill ad. Could you please provide a fake ad example here, it's easier for us to test your code, and also easier to locate possible regression in the future. Thanks.

Reply:
I set it to always be displayed.
Please note that in order to display it, it's necessary to set browser as mobile User-Agent.